test/commentLikes.spec.js

```diff
@@ -61,15 +61,15 @@ describe('User can like a comment test', () => {
           done();
         });
     });
-    it('should unlike a reply', (done) => {
+    it('should resetReaction a reply', (done) => {
```

Comment:
change _resetReaction_ to _remove reaction_ . This will make your test description clear.